flatten/flatten.py

```diff
@@ -0,0 +1,28 @@
+import unittest
+def flatten(list_a,list_b,max_depth):
```

Comment:
If we would leave a space after , the code would be easier to read and it is also recommended by pep8